hpedockerplugin/hpe/hpe_3par_mediator.py

```diff
@@ -1033,101 +588,14 @@ def create_vfs(self, vfs_name, ip, subnet, cpg=None, fpg=None,
         finally:
             self._wsapi_logout()
 
-    def set_ACL(self, fMode, fUserId, fUName, fGName):
```

Comment:
Where is this code moved?

Reply:
This was an inadvertent removal while removing unused code. I've put it back